tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "esModuleInterop": true,
+        "target": "es6",
+        "moduleResolution": "node",
+        "sourceMap": true,
+        "outDir": "dist",
+        "allowSyntheticDefaultImports": true,
+        "baseUrl": "./",
+        "strict": true,
+        "noImplicitAny": false,
+        "allowJs": true
```

Comment:
when testing building I noted it's not currently outputting declarations. to have it do so `"declaration": true` needs to be added here